scanner/scanner.go

```diff
@@ -275,6 +288,12 @@ func (s *Scanner) scanDir(tx *db.DB, c *Context, musicDir string, absPath string
 	var tracks []string
 	var cover string
 	for _, item := range items {
+		fullpath := filepath.Join(absPath, item.Name())
+		if s.excludePattern != nil && s.excludePattern.MatchString(fullpath) {
+			log.Printf("skipping excluded path `%s`", fullpath)
```

Comment:
very minor nit - just mirroring the previous log message 

```suggestion
			log.Printf("excluding path `%s`", fullpath)
```